app/views/course_assigns/_new_mobile.html.erb

```diff
@@ -32,13 +32,13 @@
       <% end %>
       <ul class="overflow-y-auto p-1 flex flex-col gap-4 h-[400px]">
         <% @courses.each do |course| %>
-          <%= render "course_assigns/course_card_mobile", course: course, f: form %>
+          <%= render 'course_assigns/course_card_mobile', course: course, f: form %>
         <% end %>
       </ul>
-      <div id="mobileFooter" class="bg-white bottom-0">
-        <%= form.button 'Assign Course', class: "bg-primary rounded w-full p-4 text-white text-base font-medium" %>
-        <%= link_to 'Cancel', request.referer, class: "text-center w-full p-4 text-base font-medium" %>
+      <div id="mobileFooter" class="bg-white bottom-0 flex flex-col items-center">
```

Comment:
- [ ] rename mobileFooter  in  app/views/course_assigns/_new_mobile.html.erb and elsewhere to mobile-footer

Reply:
Do we need id="mobileFooter" ?

Reply:
id removed

Reply:
id removed